src/Microsoft.Health.Fhir.Core/Configs/ExportConfiguration.cs

```diff
@@ -11,5 +13,10 @@ public class ExportConfiguration
         /// Determines whether export is enabled or not.
         /// </summary>
         public bool Enabled { get; set; }
+
```

Comment:
I wonder if this needs to be explicit. E.g., if we have an interface and list of providers that implement the interface, then we can query the interface and see what detination it supports and it can be automatically registered through IoC container as opposed to register the code in IoC container and also update the settings to enable it.